app/views/admin/loans/timeline/table/_step.html.slim

```diff
@@ -0,0 +1,7 @@
+- wrapper_tag = {tag: is_first ? 'div' : 'tr'}
+
+*wrapper_tag
+  / Add spacer for step who has a group as a sibling
+  - if !siblings_are_all_steps
+      td colspan="#{max_depth - entry.depth}" data-item="spacer"
```

Comment:
BTW it's fine and better to use classes instead of `data-...` if you need to identify something only for styling.

When I say to prefer to not use classes, I just mean don't use them if you could reasonably select the element some other way (for instance if it's the only `img` tag inside an outer div).

Does that make sense?